lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -736,7 +726,8 @@ public void forceDeleteTask(String taskId)
     return affectedTaskIds;
   }
 
-  public void refreshPriorityAndDueDate(String taskId) throws ClassificationNotFoundException {
+  public void refreshPriorityAndDueDateOnClassificationUpdate(String taskId)
```

Comment:
This is only used from the refresh job, isn't it? Does it need to be public for this? 
Would it be possible to apply our bulk scheme here as well?

Reply:
Needs to be public because the refresh job belongs to another package (pro.taskana.common.internal.jobs).
What do you mean with the 'bulk scheme'?

Reply:
Movement of classification update to bulk actions is subject to TSK-442, Pull request https://github.com/Taskana/taskana/pull/973